Tests/AblyChatTests/Mocks/MockRealtimePresence.swift

```diff
@@ -1,17 +1,26 @@
 import Ably
 import AblyChat
 
-final class MockRealtimePresence: RealtimePresenceProtocol {
-    var syncComplete: Bool {
-        fatalError("Not implemented")
+final class MockRealtimePresence: NSObject, @unchecked Sendable, RealtimePresenceProtocol {
```

Comment:
If it's `@unchecked Sendable` I'd expect to see some synchronisation code in here.

Reply:
addressed